csharp/src/Drivers/BigQuery/readme.md

```diff
@@ -42,17 +42,23 @@ The following parameters can be used to configure the driver behavior. The param
 
 https://cloud.google.com/dotnet/docs/reference/Google.Cloud.BigQuery.V2/latest/Google.Cloud.BigQuery.V2.QueryOptions#Google_Cloud_BigQuery_V2_QueryOptions_AllowLargeResults
 
+**adbc.bigquery.billing_project_id**<br>
+&nbsp;&nbsp;&nbsp;&nbsp;The [Project ID](https://cloud.google.com/resource-manager/docs/creating-managing-projects) used for accessing billing BigQuery. If not specified, will default to the detected project ID.
+
 **adbc.bigquery.client_id**<br>
 &nbsp;&nbsp;&nbsp;&nbsp;The OAuth client ID. Required for `user` authentication.
 
 **adbc.bigquery.client_secret**<br>
 &nbsp;&nbsp;&nbsp;&nbsp;The OAuth client secret. Required for `user` authentication.
 
+**adbc.bigquery.client.timeout**<br>
+&nbsp;&nbsp;&nbsp;&nbsp;Optional. Sets the timeout (in seconds) for the BigQueryClient. Similar to a ConnectionTimeout in ODBC.
```

Comment:
It's a bit of a nitpick, but ODBC doesn't actually have a `ConnectionTimeout` or `CommandTimeout`. These are properties surfaced on the BCL's managed wrapper for ODBC. The ODBC standard is to set attributes on the connection and statement, respectively, with attribute identifiers of `SQL_ATTR_CONNECTION_TIMEOUT` and `SQL_ATTR_QUERY_TIMEOUT`.

As such, the analogy as written isn't necessarily helpful for someone familiar with ODBC.

Reply:
fixed it here and in the readme for the BigQuery test project